modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/TableRowSkinTest.java

```diff
@@ -113,7 +252,7 @@ private void invisibleColumnsShouldRemoveCorrespondingCellsInRowImpl() {
 
     private void removedColumnsShouldRemoveCorrespondingCellsInRowImpl() {
         // Remove the last 2 columns.
-        tableView.getColumns().remove(tableView.getColumns().size() - 1, tableView.getColumns().size());
+        tableView.getColumns().remove(tableView.getColumns().size() - 2, tableView.getColumns().size());
```

Comment:
Note: I added this test in PR: https://github.com/openjdk/jfx/pull/444. While testing I found out that this removes just one column, therefore I fixed it here right away.